src/lib/__tests__/server-utilities.test.js

```diff
@@ -0,0 +1,18 @@
+import { md5Hash } from 'lib/server-utilities';
+jest.unmock('lib/server-utilities');
+
+describe('md5Hash', () => {
+  const adminClientHash = md5Hash('./static/build/editor-client.js');
+  it('returns a string', () => {
+    expect(typeof adminClientHash).toBe('string');
+  });
+
+  it('hashes consistently', () => {
+    const secondAdminClientHash = md5Hash('./static/build/editor-client.js');
+    expect(adminClientHash).toBe(secondAdminClientHash);
+  });
```

Comment:
That makes sense. In that case, it looks good to me. 👍 

Another test to ensure that the hashing is working well (or maybe an alternative to this, up to you) is testing that the MD5 hash of the `editor-client.js` file and a hash of some string like `abc` **aren't** the same.

Also, another good test would be testing that the actual string `./static/build/editor-client.js` isn't being hashed, but rather the file that the path references is what is being hashed. The mistake is easier than it sounds if the Node.js filesystem package isn't used correctly; it caused me a little headache when I was first writing this feature. This could be tested by importing the `crypto` package here and using the MD5 hash function with the string `./static/build/editor-client.js` passed in directly rather than using the one implemented in `server-utilities`.